### PR TITLE
Removed duplicate Privacy Policy

### PR DESCRIPTION
**Pull request by KimberlyMunoz**
The privacy policy is now being served by a page in CMS. This removes the duplicate hard-coded page.
I had @sarahsimpson09 confirm the correct page we want to serve.

## Removals
- Removed duplicate Privacy Policy

## Review
- @anselmbradford 
- @sebworks

### CODE REVIEW

File: src/_includes/macros/subscribe.html [hunk 1]
```diff
@@ -32,7 +32,7 @@
     <p class="short-desc u-mt10">
         The information you provide will permit the Consumer Financial
         Protection Bureau to process your request or inquiry.
-        <a href="/privacy-policy/">See More</a>
+        <a href="/sub-page/privacy-policy/">See More</a>
```
Comment (anselmbradford) [inline src/_includes/macros/subscribe.html]: `/sub-pages/privacy-policy/`

File: test/unit_tests/fixtures/forms.html [hunk 2]
```diff
@@ -9,7 +9,7 @@
     <p class="short-desc u-mt10">
         The information you provide will permit the Consumer Financial
         Protection Bureau to process your request or inquiry.
-        <a href="/privacy-policy/">See More</a>
+        <a href="/sub-page/privacy-policy/">See More</a>
```
Comment (anselmbradford) [inline test/unit_tests/fixtures/forms.html]: `/sub-pages/privacy-policy/`

Comment (KimberlyMunoz) [inline test/unit_tests/fixtures/forms.html]: Ack. Thanks for the catch!

Comment (KimberlyMunoz) [inline test/unit_tests/fixtures/forms.html]: Fixed in 1c7d275!

File: src/_includes/macros/subscribe.html (hunk 1 above)
Comment (KimberlyMunoz) [inline src/_includes/macros/subscribe.html]: Fixed in 1c7d275!

**Comment by anselmbradford** [comment]
:+1:

**Comment by jimmynotjim** [comment]
:+1:

**Pull request merged**